client/api/MovieApi.js

```diff
@@ -8,4 +8,15 @@ export function getTopMovies() {
     let promise = apiClient.get('/getTopMovies/');
     const dataPromise = promise.then((response) => response.data);
     return dataPromise;
+}
+
+export function searchMovies(query) {
+    // headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
```

Comment:
fair enough yup. As a note, you can do some cool stuff with axios to add these as defaultHeaders, but definitely way outside the scope of what we're doing here 🙌